frontend/src/widgets/OverviewWidgetField.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Tooltip as TrussWorksToolTip } from '@trussworks/react-uswds';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faQuestionCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import './OverviewWidgetField.scss';
+
+import Tooltip from '../components/Tooltip';
+import colors from '../colors';
+
+export function OverviewWidgetField({
+  label1,
+  label2,
+  route,
+  data,
+  icon,
+  iconColor,
+  backgroundColor,
+  showTooltip,
+  tooltipText,
+  filterApplicable,
+  iconSize,
+}) {
+  const smallIconClass = iconSize === 'sm' ? 'smart-hub--dashboard-overview-widget-field-icon__background-sm' : 'smart-hub--dashboard-overview-widget-field-icon__background-lg';
+  return (
+    <Grid gap={4} desktop={{ col: 'fill' }} tablet={{ col: 5 }} mobileLg={{ col: 12 }} className="smart-hub--dashboard-overview-widget-field display-flex bg-white shadow-2 padding-y-2 padding-x-1">
+      <span className="smart-hub--dashboard-overview-widget-field-icon flex-1 display-flex flex-justify-center flex-align-center">
+        <span className={`${smallIconClass} smart-hub--dashboard-overview-widget-field-icon display-flex flex-justify-center flex-align-center`} style={{ backgroundColor }}>
+          <FontAwesomeIcon color={iconColor} icon={icon} size={iconSize} />
+        </span>
+      </span>
+      <span className="smart-hub--dashboard-overview-widget-field-label display-flex flex-2 flex-column flex-justify-center">
+        <div>
+          <span className="text-bold smart-hub--overview-widget-field-data-font-size">{data}</span>
+          { !filterApplicable && (
+            <>
+              <span className="smart-hub--overview-widget-field-font-size margin-right-1"> - Filters not applied</span>
+              <TrussWorksToolTip className="usa-button--unstyled smart-hub--overview-tool-tip" id="filter-not-applicable" label="One or more of the selected filters cannot be applied to this data.">
+                <FontAwesomeIcon icon={faQuestionCircle} className="margin-left-1" color={colors.ttahubMediumBlue} />
```

Comment:
this has a classname of margin-left-1 but you set it to margin: 0 in the CSS file, I think removing them both gets you the exact same result

Reply:
Removed both.